client/onboarding/requirements/index.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import IndividualRequirements from './individual';
+import CompanyRequirements from './company';
+
+interface RequirementsProps {
+	type: 'individual' | 'company' | 'non_profit';
```

Comment:
It's also possible to have `government_entity` here (only if the country is US).

Reply:
Addressed in c5a7914